src/LiveComponent/assets/src/live_controller.ts

```diff
@@ -506,19 +506,19 @@ export default class extends Controller {
         return loadingDirectives;
     }
 
-    _showElement(element: HTMLElement) {
+    _showElement(element: HTMLElement|SVGElement) {
         element.style.display = 'inline-block';
     }
 
-    _hideElement(element: HTMLElement) {
+    _hideElement(element: HTMLElement|SVGElement) {
         element.style.display = 'none';
     }
 
-    _addClass(element: HTMLElement, classes: string[]) {
+    _addClass(element: HTMLElement|SVGElement, classes: string[]) {
         element.classList.add(...combineSpacedArray(classes));
     }
 
-    _removeClass(element: HTMLElement, classes: string[]) {
+    _removeClass(element: HTMLElement|SVGElement, classes: string[]) {
```

Comment:
Below, inside `onBeforeElUpdated()`, morphom documents that both `fromEl` and `toEl` will be `HTMLElement`. Apparently that is a lie (I just tested, if you have an svg element, it is, of course, an `SVGElement`). 

So, right inside the `if (fromEl.isEqualNode(toEl)) {` check, on the first line (before we call `cloneHTMLElement`), we should add:

```js
if (!(fromEl instanceof HTMLElement) || !(toEl instanceof HTMLElement)) {
    return false;
} 
```

In other words, if we are not dealing with HTMLElements, then assume that the elements are unchanged and do not re-render them (at this point the `isEqualNode()` has already done most checks for equality).